Content.Shared/Access/Components/AccessOverriderComponent.cs

```diff
@@ -0,0 +1,97 @@
+using Content.Shared.Access.Systems;
+using Content.Shared.Containers.ItemSlots;
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.List;
+
+namespace Content.Shared.Access.Components;
+
+[RegisterComponent, NetworkedComponent]
+[Access(typeof(SharedAccessOverriderSystem))]
+public sealed class AccessOverriderComponent : Component
+{
+    public static string PrivilegedIdCardSlotId = "AccessOverrider-privilegedId";
+
+    [DataField("privilegedIdSlot")]
+    public ItemSlot PrivilegedIdSlot = new();
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("denialSound")]
+    public SoundSpecifier? DenialSound;
+
+    public EntityUid TargetAccessReaderId = new();
+
+    [Serializable, NetSerializable]
+    public sealed class WriteToTargetAccessReaderIdMessage : BoundUserInterfaceMessage
+    {
+        public readonly List<string> AccessList;
+
+        public WriteToTargetAccessReaderIdMessage(List<string> accessList)
+        {
+            AccessList = accessList;
+        }
+    }
+
+    [DataField("accessLevels", customTypeSerializer: typeof(PrototypeIdListSerializer<AccessLevelPrototype>))]
+    public List<string> AccessLevels = new()
+    {
+        "Armory",
```

Comment:
Any chance this could be built at start after reading in access level prototypes? Some forks may have different jobs, and jobs may be added via a YAML change.

Reply:
That's definitely doable. Someone should also look at the code for the ID card console as well then, since that's what I based it on